packages/mobile/src/identity/verification.ts

```diff
@@ -64,6 +68,13 @@ export interface AttestationCode {
   issuer: string
 }
 
+export function* isSufficientBalance() {
+  const e164Number: string = yield select(e164NumberSelector)
+  const cachedSalt = yield call(getSaltCache, e164Number)
```

Comment:
Use the `getUserSelfPhoneHashDetails` function in privateHashing so we can keep all interaction with these salt related states exlcusively in the privateHashing.ts file